client/look-and-feel/css/src/Form/Select/Select.scss

```diff
@@ -74,5 +74,9 @@
         box-shadow: 0 0 0 1px var(--color-red-700) inset;
       }
     }
+
+    &--filled:not(:hover, :focus, :active) {
```

Comment:
je pense que tu peux éviter de créer une classes css : tu peux utiliser le :not(:empty))

https://developer.mozilla.org/en-US/docs/Web/CSS/:empty
(attention au white space)